specs/lightclient/beacon-chain.md

```diff
@@ -69,7 +69,8 @@ This is a patch implementing the first hard fork to the beacon chain, tentativel
 This is formatted as an enum, with values `2**i` that can be combined as bit-flags.
 The `0` value is reserved as default. Remaining bits in `ValidatorFlags` may be used in future hardforks.
 
-**Note**: unlike Phase0, a `TIMELY_TARGET_FLAG` does not imply a `TIMELY_SOURCE_FLAG`.
+**Note**: Unlike Phase0, a `TIMELY_TARGET_FLAG` does not necessarily imply a `TIMELY_SOURCE_FLAG`
+due to the varying slot delay requirements of each.
 
```

Comment:
I didn't understand here, their assignements are by
```
    if is_matching_source and state.slot <= data.slot + integer_squareroot(SLOTS_PER_EPOCH):
        participation_flags.append(TIMELY_SOURCE_FLAG)
    if is_matching_target and state.slot <= data.slot + SLOTS_PER_EPOCH:
        participation_flags.append(TIMELY_TARGET_FLAG)
```
And `SLOTS_PER_EPOCH`  is always larger or equal than its square_root